scapy/contrib/automotive/someip.py

```diff
@@ -218,6 +218,8 @@ def _bind_someip_layers():
     for i in range(15):
         bind_layers(UDP, SOMEIP, sport=30490 + i)
         bind_layers(TCP, SOMEIP, sport=30490 + i)
+        bind_layers(UDP, SOMEIP, dport=30490 + i)
+        bind_layers(TCP, SOMEIP, dport=30490 + i)
```

Comment:
When you bind things for dissection, use
`bind_bottom_up`
Then you can use a single
`bind_layers(UDP, SOMEIP, dport=[..], sport=[..])` (or `bind_top_down`)
which will be the default for building

Reply:
fixed